packages/@webex/plugin-meetings/src/interceptors/locusRetry.ts

```diff
@@ -0,0 +1,67 @@
+/*!
+ * Copyright (c) 2015-2020 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {Interceptor} from '@webex/http-core';
+
+const rateLimitExpiryTime = new WeakMap();
+/**
+ * @class
+ */
+export default class LocusRetryStatusInterceptor extends Interceptor {
+  /**
+   * @returns {LocusRetryStatusInterceptor}
+   */
+  static create() {
+    // @ts-ignore
+    return new LocusRetryStatusInterceptor({webex: this});
+  }
+
+  /**
+   * Handle response errors
+   * @param {Object} options
+   * @param {WebexHttpError} reason
+   * @returns {Promise<WebexHttpError>}
+   */
+  onResponseError(options, reason) {
+    if (reason.statusCode >= 500 && reason.statusCode < 600 && options.uri.includes('locus')) {
```

Comment:
I see this here, but why wouldn't this be parameterized?

Also shouldn't this be specific to 503? not 5xx ?

Reply:
This could be the case, the JIRA appears to read that it needs to be specifically 503.